source/extensions/compression/gzip/decompressor/config.cc

```diff
@@ -9,11 +9,15 @@ namespace Decompressor {
 namespace {
 const uint32_t DefaultWindowBits = 12;
 const uint32_t DefaultChunkSize = 4096;
+// When summed to window bits, this tells zlib library to decompress gzip data per:
```

Comment:
nit: s/summed/logical ORd or something like that (even though sum is technically correct I think this is more clear)